src/components/campaigns/CampaignsList.tsx

```diff
@@ -7,6 +7,10 @@ import CampaignCard from './CampaignCard'
 
 export default function CampaignsList() {
   const { data, isLoading } = useCampaignList()
+  let mobile = false
+  if (/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)) {
+    mobile = true
+  }
```

Comment:
We better use the standard MUI approach to the responsive sizes and get it via the `useMediaQuery()` hook

```
import { useMediaQuery } from '@mui/material'
```

```suggestion
  const mobile = useMediaQuery('(max-width:900px)')
```